Accept applicant page

tanaya09 opened the pull request:
<!-- Try to fill out everything! -->

## What does this pull request include?
1. UI for accept applicant
<!-- List the features you did -->

## Any outstanding issues in this pull request?
No
<!-- List anything you want others to fix -->

## Checklist

- [x] Test locally
- [x] Assign a reviewer
- [ ] ZenHub

austinhuang0131 reviewed (changes requested):
should also reveal their email address for contact (this was mentioned at the end of last demo), same should also be done for the applicant

otherwise ok

tanaya09 commented:
I’ve addressed the requested changes:
Improved the confirmation UI for the applicant page.
Added an email display for the selected applicant. 
Please let me know if further adjustments are needed.

austinhuang0131 commented on Time_Banking/Time_Banking/templates/profile_info.html:
```diff
@@ -20,11 +20,17 @@
                     <p><i class="fa fa-map-marker"></i> {{ user.location }}</p>
                     <p><i class="fa fa-calendar"></i> Joined in {{ user.date_joined|date:"F Y" }}</p>
                     <p>Bio: {{ user.bio }}</p>
+<p><i class="fa fa-envelope"></i> {{ user.email }}</p>
```
not necessary, my point was to only provide email after being accepted, but the person who applied to the service should be able to see the email of the provider on applied_services as well

austinhuang0131 commented on Time_Banking/Time_Banking/templates/view_applicants.html:
```diff
@@ -19,11 +19,14 @@ <h4 class="confirmation-title">Selection confirmed successfully!</h4>
                     </p>
+                    <p class="confirmation-detail">
+                        <i class="fa fa-envelope"></i> {{ response.user.email }}
+                    </p>
```
ok
